.travis.yml

```diff
@@ -8,11 +8,6 @@ php:
 
 sudo: false
 
-env:
```

Comment:
No problem, I did this in regards to having to change the keys in general anyways per @omgitstom 's request to all of us to keep testing within the same tenant